### ISSUE TITLE
Failed to find matching world for 'Mond'

### ISSUE THREAD

**Issue by dredhorse** [micdoodle8/Galacticraft#2222]
Mond is german for moon.
    # Dimension ID for the Moon
    I:idDimensionMoon=-28

this looks like it exists

at least DIM-28 is in the world directory

also running Galaxy Space. This is unabridged mod pack R11

GC 3.0.12.446
GS 1.0.8STABLE

**Comment by dredhorse** [comment]
```
[22:09:30] [Server thread/INFO] [Galacticraft/]: Failed to find matching world for 'Mond'
[22:09:30] [Server thread/ERROR] [Galacticraft/]: Error occurred when attempting to transfer entity to dimension: Mond
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: java.lang.NullPointerException
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at micdoodle8.mods.galacticraft.core.network.PacketSimple.handleServerSide(PacketSimple.java:853)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at micdoodle8.mods.galacticraft.core.network.GalacticraftPacketHandler.channelRead0(GalacticraftPacketHandler.java:27)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at micdoodle8.mods.galacticraft.core.network.GalacticraftPacketHandler.channelRead0(GalacticraftPacketHandler.java:12)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:98)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:785)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.embedded.EmbeddedChannel.writeInbound(EmbeddedChannel.java:169)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.common.network.internal.FMLProxyPacket.func_148833_a(FMLProxyPacket.java:77)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
[22:09:30] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
```

**Comment by dredhorse** [comment]
Ok, I found the issue.

You are checking for the NAME which is passed from the client, the problem is that the client is running in  German and the dedicated server defaults to the english language.

So you need to rewrite client / server interaction to pass a token between client and server. 

@BlesseNtumble added you because I don't know how much you use the code of GalactiCraft or use the same approach.

**Comment by BlesseNtumble** [comment]
Download 454 or higher build GC3

**Comment by radfast** [comment]
Duplicate of #2141 
This was quickly fixed, has been fixed ever since Galacticraft 3.0.12.451
